mne/io/kit/constants.py

```diff
@@ -167,6 +179,7 @@
     KIT.SYSTEM_UMD_2014_07: None,
     KIT.SYSTEM_UMD_2014_12: 'KIT-UMD-3',
     KIT.SYSTEM_UMD_2019_09: None,
+    KIT.SYSTEM_YOKOGAWA: None,
```

Comment:
@apesquita @Kyung-minAn can you help us here? what can you tell us about the MEG system name/version/location?

Reply:
@agramfort , I'm asking to KIT about this issue.
I will let you know as soon as I get this info.
Thank you for developing the reader~

Reply:
@agramfort I contacted KIT for this issue.
I heard that It might take some days for the person in charge to check e-mail because it is the holiday season in Japan.

Reply:
@agramfort,
I asked KIT and Ricoh company about this information.
They provided information about each MEG system (name/version/location).
SYSTEM_UMD_2014_07 :

Reply:
@Kyung-minAn what would be great would be to have a tiny file from each of these YOKOGAWA systems. Do you think it's possible?

Reply:
@agramfort 
Do you mean three Yokogawa systems?

Reply:
It would be good to have a file from each of the three Yokogawa systems if possible, yes. Then we can test that these values are used/set correctly by us

Reply:
@agramfort @larsoner 
I appreciate your great contributions.
The short data that I sent before is recorded from the child-customized Yokogawa MEG system (PQ1151R).
For two Yokogawa adult MEG systems, I will ask Ricoh company to provide the data.
Do you have any preference for the testing data? Would it be fine with each coordinated con file (1-s long) and MRI image?
Thanks a lot~!!